inc/form.class.php

```diff
@@ -662,13 +662,17 @@ public function showWizard() : void {
       $sort_order = PluginFormcreatorEntityconfig::getUsedConfig('sort_order', Session::getActiveEntity());
       $selected = $sort_order == PluginFormcreatorEntityconfig::CONFIG_SORT_POPULARITY ? 'checked="checked"' : '';
       echo '<input type="radio" class="-check-input" id="plugin_formcreator_mostPopular" name="sort" value="mostPopularSort" '.$selected.' onclick="showTiles(tiles)"/>';
-      echo '<label for="plugin_formcreator_mostPopular">&nbsp;'.$sort_settings[PluginFormcreatorEntityConfig::CONFIG_SORT_POPULARITY] .'</label>';
+      echo '<label for="plugin_formcreator_mostPopular">';
+      echo '<a><i class="fa fa-star" aria-hidden="true"></i></a>';
```

Comment:
We should add a tooltip on the anchor 

```suggestion
      echo '<a title="' . $sort_settings[PluginFormcreatorEntityConfig::CONFIG_SORT_POPULARITY] . '"><i class="fa fa-star" aria-hidden="true"></i></a>';
```